src/main/java/com/shaft/gui/browser/BrowserActions.java

```diff
@@ -21,6 +22,7 @@
     private static final Boolean HEADLESS_EXECUTION = Boolean.valueOf(System.getProperty("headlessExecution").trim());
     private static final int NAVIGATION_TIMEOUT_INTEGER = Integer
             .parseInt(System.getProperty("browserNavigationTimeout").trim());
+    private static final Duration NAVIGATION_TIMEOUT = Duration.ofSeconds(NAVIGATION_TIMEOUT_INTEGER);
```

Comment:
Codacy found an issue: [Avoid unused private fields such as 'NAVIGATION_TIMEOUT'.](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5771854)

Reply:
in place for easy upgrade to selenium 4.x